UnitsNet/UnitDefinitions/AreaDensity.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "Name": "AreaDensity",
+  "BaseUnit": "KilogramPerMeterSquared",
+  "XmlDoc": "The area density of a two-dimensional object is calculated as the mass per unit area.",
+  "Units": [
+    {
+      "SingularName": "KilogramPerMeterSquared",
```

Comment:
Please use `SquareMeter` instead of instead of `MeterSecondSquare` to be consistent with other units

Reply:
Good point, we will modify as requested.